### PR TITLE
setup gateway authorization

### PR DESCRIPTION
**Pull request by isaev-co**
PR for FE: https://github.com/ConstantineIsaev/vinyl-store-fe/pull/6
(also added alerts for the responses in 401 and 403 HTTP statuses as optional task)

Client APP in cloudFront: https://d111triofks8tr.cloudfront.net/admin/products
for testing: provide authorization_token property in localStorage with valid value Q29uc3RhbnRpbmVJc2FldjpURVNUX1BBU1NXT1JE (ConstantineIsaev:TEST_PASSWORD)

expected score 6/6

### CODE REVIEW

**Comment by kotmatpockuh** [comment]
Don't forget to update the PR description in time ;)

File: import-service/serverless.yml [hunk 1]
```diff
@@ -32,6 +53,12 @@ functions:
           path: import
           method: get
           cors: true
+          authorizer:
+            name: tokenAuthorizer
+            arn: "arn:aws:lambda:us-east-1:241876787615:function:authorization-service-dev-basicAuthorizer"
```
Comment (evgenii-alekseenko) [inline import-service/serverless.yml:58]: Hardcode? I'd recommend to obtain this `arn` programmatically. There are at least couple of ways how to refer to this lambda =)

**Review by evgenii-alekseenko — APPROVED** [review]

File: import-service/serverless.yml (hunk 1 above)
Comment (skyRoma) [reply import-service/serverless.yml:58]: Agree, for example:
image

File: authorization-service/src/basicAuthorizer.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+module.exports = async (event, ctx, cb) => {
+  if (event['type'] != 'TOKEN') {
```
Comment (skyRoma) [inline authorization-service/src/basicAuthorizer.js:2]: Why not just `event.type`? :)

**Review by skyRoma — APPROVED** [review]

**Pull request merged**